stack.yaml

```diff
@@ -1,8 +1,14 @@
-resolver: nightly-2019-02-17
+resolver: lts-17.0
```

Comment:
Any reason to not use the latest available which is https://www.stackage.org/lts-17.9?

Reply:
Done.